### PR TITLE
Nisha | Test | subscription responsive new

### PR DESCRIPTION
**Pull request by nisha-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss [hunk 1]
```diff
@@ -23,11 +23,23 @@
...
+
+            span {
+                overflow: hidden;
+                display: -webkit-box;
+                -webkit-box-orient: vertical;
+                -webkit-line-clamp: 2;
+            }
+
+            @media screen and (max-width: 1299px) {
+                font-size: 18px !important;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss]: why use static font ?

We have available font sizes

 --font-size-16 and --font-size-20 :

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss [hunk 2]
```diff
@@ -40,6 +52,14 @@
             s {
                 color: var(--color-gray);
             }
+
+            @media screen and (max-width: 1299px) {
+                margin-bottom: 6px !important;
+
+                .plan-price {
+                    font-size: 18px !important;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss]: why use static font ?

We have available font sizes

--font-size-16 and --font-size-20

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss]: done

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.scss]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**